Test Grafana Backup and Restore (#3211)
kyma-project/kyma: resources/ark/README.md

type BackupTest interface {
    CreateResources(namespace string)
    TestResources(namespace string)
    DeleteResources()
}
```
The functions work as follows:

- The `TestResources` function validates if the test data works as expected. 
- The `CreateResources` function installs the required test data before the backup process starts.

> - The `DeleteResources` function deletes those resources that are  part of the cluster before the execution of the test and need to be deleted to test the restore process.

Reviewer: ```suggestion
- The `DeleteResources` function deletes the resources that are a part of the cluster before executing the test. The resources need to be deleted to test the restore process.
```
